unit-tests/live/frames/test-fps-permutations.py

```diff
@@ -0,0 +1,304 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:donotrun:!nightly
+# test:timeout 750
+# timeout - on the worst case, we're testing on D585S, which have 8 sensors, so:
+# timeout = (8 choose 2) * 24 + 24 = 696
+# 8 choose 2 tests to do (one for each pair), plus one for all sensors on, each test takes 24 secs
+
+from rspy import test, log
+import time
+import itertools
+import math
+
+
+# To add a new test mode (ex. all singles or triplets) only add it below and on should_test
+# add it in get_time_est_string for an initial time calculation
+# run perform_fps_test(sensor_profiles_array, [mode])
+ALL_PERMUTATIONS = 1
+ALL_PAIRS = 2
+ALL_SENSORS = 3
+
+# global variable used to count on all the sensors simultaneously
+count_frames = False
+
+# tests parameters
+TEST_ALL_COMBINATIONS = False
+seconds_till_steady_state = 4
```

Comment:
Let's align the coding style and naming convention and keep all constants as uppercase letters